.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Problem description
+
+
+
+### Screenshot / Reference 
+
+
+
+### Checklist
+- [ ] I've set a rough estimate | Estimate is TBA
```

Comment:
Probably should save this for sprint planning

Reply:
It's pretty good! So the screenshot can be for the bug or for the new design?

Reply:
@tkeuneman Yeah a reference of some kind, it could be a screenshot of the UI bug, new design or the suspected file the bug might be in. I'll update the heading to `Screenshot / Design / File reference` ? Not sure if that make it's clear for somebody...